### ISSUE TITLE
Drop the `eks.` subdomain from `govuk.digital.` domains

### ISSUE THREAD

**Issue by theseanything** [alphagov/govuk-infrastructure#1280]
Very low priority enhancement. As we no longer are two sets of infrastructure in parallel, having the `eks.` subdomain for our  `govuk.digital.` domains isn't very useful. We don't need to distinguish between sets of infrastructure, nor do we really need to know that it's host on EKS.

We could simplify domains for example:

`argo.eks.integration.govuk.digital` -> `argo.integration.govuk.digital`

Saves a few extra keystrokes when filling in a URL (particularly before autocomplete kicks in for the environment).

Realise that this might be high effort/risk vs value, due to things like Dex callback URLs, CDN config etc etc.

**Comment by samsimpson1** [comment]
[backlog refinement] we've decided to not do this work, since it doesn't get us many benefits for the risk/work required
